react-toggle-switch-component/src/toggle-switch.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+let clickCount = 0;
+
+class ToggleSwitch extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { isClicked: false };
+  }
+
+  handleClick() {
+    this.setState({ isClicked: true });
+  }
+
+  render() {
+    clickCount++;
+    if (clickCount % 2 === 0) {
+      return (
+        <div>
+          <div className="button-container">
+            <button onClick={this.handleClick.bind(this)} className="on"></button>
+            <div onClick={this.handleClick.bind(this)} className="on-circle"></div>
+          </div>
+          <p>ON</p>
+        </div>
+      );
+    } else if (clickCount % 2 === 1) {
```

Comment:
Just an `else` here would work, you don't need to check the opposite of what was already checked.